### ISSUE TITLE
Add names to the idol filter

### ISSUE THREAD

**Issue by summertriangle-dev** [SchoolIdolTomodachi/SchoolIdolAPI#259]
The chibi list is kinda hard to read.

![blah](https://cloud.githubusercontent.com/assets/7821037/19009199/0c65f148-8726-11e6-8cf6-0e6400744c6b.png)

**Comment by neokai82** [comment]
Agreed, but your suggestion would make finding the minor characters harder - would you know off-hand the name of A-Rise members?

**Comment by pighair** [comment]
It's really hard to find non-μ’s and non-Aqours members from chibi. If the list displays name under chibi, it'll be much easier to find one.
